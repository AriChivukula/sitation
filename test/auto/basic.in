See, e. g., Birchem v. Knights of Columbus, 116 F.3d 310, 312-313 (CA8 1997); cf. Nationwide Mut. Ins. Co. v. Darden, 503 U. S. 318, 322323 (1992); Id. at 435
