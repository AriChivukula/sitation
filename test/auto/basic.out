10,See, e.g.,,116,F.3d,310,312-313,1
12,Cf.,503,U.S.,318,322323;1992,1
1,,0,,0,,0
